keyboards/fr4/unix60/readme.md

```diff
@@ -0,0 +1,16 @@
+# Unix60
+
+![unix60](https://raw.githubusercontent.com/mkdl/Unix60/main/Pictures/Top.JPG)
```

Comment:
Please rehost this on imgur, and add `l` to the end of the filename to make it thumbnail sized.

Reply:
done